content/docs/integrations/aws-resources/index.md

```diff
@@ -10,27 +10,80 @@ integrationType: Catalog data source
 
 ## Introduction
 
-Roadie has the capability to automatically ingest resources from AWS. This is done by automatic discovery and currently ingests all configured resource types per AWS region.
+Roadie has the capability to automatically ingest resources from AWS. This is done by automatic discovery and currently ingests all configured resource types per AWS region. You can configure AWS resource ingestion either by manually adding standalone account/region configurations or by using AWS organization structure to autodiscover accounts and their resources.
```

Comment:
This might be a bit too low level for this type of document but is it worth explaining what we mean by ingestion ? e.g. that AWS resources are represented by (Resource) entities in the catalog ?

Reply:
makes sense :+1: